src/compiler/gcc.rs

```diff
@@ -438,6 +439,11 @@ where
     if !may_dist && !parsed_args.profile_generate {
         cmd.arg("-P");
     }
+    match kind {
+       CCompilerKind::Clang => { cmd.arg("-frewrite-includes"); },
```

Comment:
nit: braces are not necessary, so probably just: `=> cmd.arg(".."),`

Reply:
That yields me:
```
error[E0308]: match arms have incompatible types
   --> src/compiler/gcc.rs:453:14
    |
450 | /     match kind {
451 | |         CCompilerKind::Clang => cmd.arg("-frewrite-includes"),
    | |                                 ----------------------------- this is found to be of type `&mut <T as mock_command::CommandCreatorSync>::Cmd`
452 | |         CCompilerKind::GCC => cmd.arg("-fdirectives-only"),
    | |                               ---------------------------- this is found to be of type `&mut <T as mock_command::CommandCreatorSync>::Cmd`
453 | |         _ => (),
    | |              ^^ expected mutable reference, found ()
454 | |     }
    | |_____- `match` arms have incompatible types
    |
    = note: expected type `&mut <T as mock_command::CommandCreatorSync>::Cmd`
               found type `()`
```

Reply:
Ah, indeed. Nevermind then :)